### PR TITLE
Public ap is

### PR DESCRIPTION
**Pull request by akshaykalucha3**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [x] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -442,6 +442,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Kwelo Network](https://www.kwelo.com/network/ip-address) | Locate and get detailed information on IP address | No | Yes | Yes |
 | [LocationIQ](https://locationiq.org/docs/) | Provides forward/reverse geocoding and batch geocoding | `apiKey` | Yes | Yes |
 | [Mapbox](https://www.mapbox.com/developers/) | Create/customize beautiful digital maps | `apiKey` | Yes | Unknown |
+| [MetaURLtags](https://promptapi.com/marketplace/) | Scrapes, parses and retrieves META tags of any URL | `apiKey` | Yes | Unknown |
```
Comment (yannbertrand) [inline README.md:445]: Looks like a marketplace of APIs. Is it the URL you wanna add?

File: README.md [hunk 2]
```diff
@@ -731,6 +732,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Fuck Off as a Service](https://www.foaas.com) | Asks someone to fuck off | No | Yes | Unknown |
 | [Full Contact](https://www.fullcontact.com/developer/docs/) | Get Social Media profiles and contact Information | `OAuth` | Yes | Unknown |
 | [HackerNews](https://github.com/HackerNews/API) | Social news for CS and entrepreneurship | No | Yes | Unknown |
+| [Hashtags](https://ritekit.com/api-demo/hashtag-suggestions) | Returns list of hashtag suggestions | `apiKey` | Yes | Unknown |
```
Comment (yannbertrand) [inline README.md:735]:
```suggestion
| [Hashtags](https://documenter.getpostman.com/view/2010712/SzS7Qku5#4f564fac-f015-4367-9831-859e00a1c673) | Hashtag related endpoints like suggestions from a text | `apiKey` | Yes | Unknown |
```

File: README.md [hunk 3]
```diff
@@ -749,6 +751,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Twitter](https://developer.twitter.com/en/docs) | Read and write Twitter data | `OAuth` | Yes | No |
 | [vk](https://vk.com/dev/sites) | Read and write vk data | `OAuth` | Yes | Unknown |
 
+
```
Comment (yannbertrand) [inline README.md:754]:
```suggestion
```

**Review by yannbertrand — CHANGES_REQUESTED** [review]

**Comment by Tiaogaviao** [comment]
Ganhar dinheiro baixando o Kwai!https://s.kwai.app/s/oenFdtwS

File: README.md (hunk 1 above)
Comment (Tiaogaviao) [reply README.md:445]: Ganhar dinheiro baixando o Kwai!https://s.kwai.app/s/oenFdtwS

File: README.md (hunk 3 above)
Comment (Tiaogaviao) [reply README.md:754]: Ganhar dinheiro baixando o Kwai!https://s.kwai.app/s/oenFdtwS

**Comment by Tiaogaviao** [comment]
https://vm.tiktok.com/ZMdXARYaw/

**Pull request closed without merging**